test/run-windows-script.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
```

Comment:
What is this script used for?

Reply:
at first, I was trying to use a simple script to deploy and run tests for windows cluster. But later I release it missed driver build part which is available in the run-integration-test go program. So this script is not really used. I can remove it or adjust it and make it available just for users

Reply:
If you want to add it in, maybe put it in an "experimental" directory and add a comment at the top explaining what the script is used for.

Reply:
I will remove it for now, and add it later if needed.